src/Alert/Alert.js

```diff
@@ -0,0 +1,235 @@
+/* @flow */
+
+import * as React from 'react';
+import { Animated } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { getStatusBarHeight } from 'react-native-status-bar-height';
+
+import StyleSheet from '../PlatformStyleSheet';
+import Text from '../Text';
+
+import type { OnLayout, NotificationType } from '../types';
+import type { StylePropType } from '../PlatformStyleSheet/StyleTypes';
+
+type Props = {|
+  +style?: StylePropType,
+|};
+
+type State = {|
+  layout: {
+    height: number,
+    measured: boolean,
+  },
+  opacity: Animated.Value,
+  translateY: Animated.Value,
+  isOpen: boolean,
+  alertType: NotificationType,
+  alertMessage: React.Node | string,
+|};
+
+const ANIMATION_DURATION = 250;
+const DISPLAY_DURATION = 1500;
+
+export default class Alert extends React.Component<Props, State> {
+  state = {
+    layout: {
+      height: -100,
+      measured: false,
+    },
+    opacity: new Animated.Value(0),
+    translateY: new Animated.Value(0),
+    isOpen: false,
+    alertType: 'error',
+    alertMessage: '',
+  };
+
+  _lastCallTimestamp = null;
+  _defferedHideAlert = undefined;
+  _hideTimeout: TimeoutID;
+
+  componentWillUnmount() {
+    clearTimeout(this._hideTimeout);
+  }
+
+  handleLayout = (e: OnLayout) => {
+    const { layout } = this.state;
+    const { height } = e.nativeEvent.layout;
+    const { measured } = layout;
+    const { isOpen, translateY, opacity } = this.state;
+
+    this.setState({ layout: { height, measured: true } }, () => {
+      if (measured) {
+        if (!isOpen) {
+          // - 300 is mainly for displaying in storybook
```

Comment:
Not fully understand this but if it's related to Storybook... It probably should not be an extra case.

Reply:
Only the value `-300` is needed for proper displaying in storybook, but it doesnt affect the component behaviour.

Reply:
Rendering `null` when `isOpen === false`